src/main.c

```diff
@@ -7,14 +7,15 @@
 
 int main()
 {
+    rcc_init_clocks();
     rcc_gpio_set(A, 1);
-    rcc_uart_set(UART_PORT_1, 1);
+    rcc_uart_set(UART_PORT_2, 1);
 
     GPIO_config PA5_config = {
         .pin = GPIO_PIN_5,
         .mode = GPIO_MODE_OUTPUT,
         .pull = GPIO_NO_PUPD,
-        .speed = GPIO_SPEED_LOW 
+        .speed = GPIO_SPEED_LOW
```

Comment:
nitpick: it's best to add trailing commas whenever possible (this produces cleaner diffs if you add another option after `.speed`)